File: robin/java/src/level1/study5/CountPY.java
package level1.study5;

public class CountPY {
    public static boolean solution(String s) {

        String lowerS = s.toLowerCase();
        char[] arrayS = lowerS.toCharArray();

        int countP = 0;
        int countY = 0;
        for (int i = 0; i < arrayS.length; i++) {
            if (arrayS[i] == 'p') countP += 1;
            if (arrayS[i] == 'y') countY += 1;
        }

        if (countP != countY) return false;
        return true;
    }

    public static void main(String[] args) {
        String s = "pPoooyY";
        System.out.println(solution(s));
    }
}

File: robin/python/programmers/level1/study5/count_p_y.py
def solution(s):
    lower_s = s.lower()
    count_p = lower_s.count('p')
    count_y = lower_s.count('y')

    if count_p == count_y:
        return True
    return False


def solution1(s):
    return s.lower().count('p') == s.lower().count('y')


if __name__ == '__main__':
    s = "pPoooyY"
    print(solution(s))
    print(solution1(s))
